jbmc/src/janalyzer/janalyzer_parse_options.cpp

```diff
@@ -701,6 +701,21 @@ void janalyzer_parse_optionst::help()
     "                              class specified in the JAR manifest\n"
     "                              (checked in this order).\n"
     "\n"
+    " -classpath dirs/jars\n"
+    " -cp dirs/jars\n"
+    " --classpath dirs/jars        set class search path of directories and\n"
+    "                              jar files\n"
+    "                              A "
+    #ifdef _WIN32
+    ";"
+    #else
+    ":"
```

Comment:
:pick: this could be put in a `java_file_separator_char` function, as it can be used in several places of jbmc